x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/properties/table.tsx

```diff
@@ -80,7 +79,7 @@ export const Table = (props: Props) => {
                 </EuiFlexItem>
                 <EuiFlexItem>
                   {!Array.isArray(item.value) && item.value}
-                  {Array.isArray(item.value) && <ArrayValue values={item.value} />}
+                  {Array.isArray(item.value) && <ExpandableContent values={item.value} />}
```

Comment:
**nit:** same as [this comment](https://github.com/elastic/kibana/pull/152956#discussion_r1144358629) explains.

Reply:
Thanks for finding that, removed the conditions ✅